smf/sled-agent/config-rss.toml

```diff
@@ -67,3 +67,12 @@ addresses = [ "fd00:1122:3344:0101::4" ]
 gz_addresses = []
 [request.service.service_type]
 type = "oximeter"
+
+[[request.service]]
+id = "a0fe5ebc-9261-6f77-acc1-972481755789"
+name = "dendrite"
+addresses = [ "fd00:1122:3344:0101::9" ]
+gz_addresses = []
+[request.service.service_type]
+type = "dendrite"
+asic = "tofino_stub"
```

Comment:
> Is it critical that the Dendrite service be up and running before Nexus, or can it be initialized after Nexus comes online?

Yes, without dendrite up the bootstrap agent won't come up and you can't get Nexus running because you won't be able to unlock the trust quorum.